build.gradle

```diff
@@ -215,8 +281,31 @@ project(':par-connector') {
         includeEmptyDirs = false
     }
 
-    task startScheduler(dependsOn: [distScheduler, installJars, reinstallRPackage]) << {
-        schedProcess = ['java', '-classpath', 'dist/lib/*', 'org.ow2.proactive.scheduler.util.SchedulerStarter', '-ln', '2', '-t', '60000'].execute(null, new File(schedHome))
+    task changeSchedulerSettings(dependsOn: distScheduler) << {
+        def schediniFile = file("${rootProject.buildDir}/scheduler/config/scheduler/settings.ini")
+        def pattern = java.util.regex.Pattern.compile("pa\\.scheduler\\.task\\.fork=.*")
```

Comment:
I don't have access to the Properties classes from pa-rconnector project, and I would like to avoid adding dependencies (as they may be by mistake wrapped in the PARConnector R package)